packages/components/src/components/SquiggleChart.tsx

```diff
@@ -23,6 +22,17 @@ export type SquiggleChartProps = {
   // `environment` is passed through StandaloneExecutionProps; this way we guarantee that it's not compatible with `project` prop
   Omit<PartialPlaygroundSettings, "environment">;
 
+// type RunSetup =
```

Comment:
I'd be interested int changing to this (also showed in Discord), but wanted to check in with you first.